docs/sources/administration/service-accounts/enable-service-accounts.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Enable service accounts in Grafana'
+menuTitle: 'Enable service accounts'
+description: 'This contains information to enable service accounts feature in Grafana'
+aliases: [docs/sources/administration/service-accounts/enable-service-accounts.md]
+weight: 40
+keywords:
+  - Feature toggle
+  - Service accounts
+---
+
+## Enable service accounts in Grafana
+
+Service accounts are available behind the `service-accounts` feature toggle in Grafana 9.0+.
+You can enable it either in a [config file]({{< relref "../../administration/configuration.md#config-file-locations" >}}) or by [configuring an environment variable]({{< relref "../../administration/configuration/#configure-with-environment-variables" >}}).
+
+### Enable service accounts with configuration file
+
+In your [config file]({{< relref "../../administration/configuration.md#config-file-locations" >}}), add `service-accounts` as a [feature_toggle]({{< relref "../../administration/configuration.md#feature_toggle" >}}).
+
+```
+[feature_toggles]
+# enable features, separated by spaces
+enable = service-accounts
+```
+
+### Enable service accounts with an environment variable
+
+You can use `GF_FEATURE_TOGGLES_ENABLE = service-accounts` environment variable to override the configuration file.
```

Comment:
Use the same approach that I described above to add an intro, before you begin, stem sentence, and steps.